data/korath/korath hails.txt

```diff
@@ -0,0 +1,1470 @@
+		"exile humanika: are above par by any reasonable standards"
+	phrase
+		"exile mixed: , you inferior [individual|species]"
+		"exile: empty phrase"
+	word
+		"."
+
+phrase "exile humanika: Your ship or its contents or attributes"
+	word
+		"Yer ship"           # Your ship (mispronounced)
+		"Yoo"                # You (mispronounced)
+		"Yer injuns"         # Your engines (mispronounced)
+		"Yu wepuns"          # Your weapons (mispronounced)
+		"'oo gen'rator"      # Your [power] generator (mispronounced)
+		"Dem ship"           # Them ship (mispronounced)
+
+phrase "exile humanika: Our ship or its contents or attributes"
+	word
+		"Are ship"           # Our ship (mispronounced)
+		"Wee"                # We (mispronounced)
+		"Mi injuns"          # My engines (mispronounced)
+		"Oar wepuns"         # Our weapons (mispronounced)
+		"Ere pasa core"      # Our plasma core (mispronounced)
+		"Mi ship"            # My ship (mispronounced)
+
+phrase "exile humanika: are sub-par by any reasonable standards"
+	word
+		"'s no gud"          # is no good (mispronounced)
+		"baad"               # bad (mispronounced)
+		"be dren"            # is dren (mispronounced)
+		"dust'n wurk"        # doesn't work (mispronounced)
+		"ain' gud"           # ain't good (mispronounced)
+
+phrase "exile humanika: are above par by any reasonable standards"
+	word
+		"'s amazin"          # is amazing (mispronounced)
+		"do gud"             # [do|does] good (mispronounced)
+		"werks wall"         # works well (mispronounced)
+		"snassy"             # [snazzy|sassy] (mispronounced)
+		"hi tek"             # [is] high tech (mispronounced)
+
+phrase "exile humanika: are inferior to what we possess"
+	word
+		"les ours"           # [is] less [than] ours (mispronounced)
+		"smalrn oars"        # [is] smaller than ours (mispronounced)
+		"week thin me"       # [is] weaker than me (mispronounced)
+
+phrase "exile humanika: are superior to what you possess"
+	word
+		"be'ren yurs"        # [is] better than yours (mispronounced)
+		"bigger yers"        # [is] bigger than yours (mispronounced)
+		"ston thin you"      # [is] stronger than [you|yours] (mispronounced)
+
+phrase "exile: empty phrase"
+	word
+		""
+
+phrase "exile mixed: , you inferior [individual|species]"
+	word
+		", "
+	phrase
+		"exile humanika: you inferior species"
+		"exile humanika: you inferior individual"
+
+phrase "exile humanika: you inferior species"
+	word
+		"yuu beeg apes"       # You big apes (mispronounced)
+		"humanika dren"       # human dren (mispronounced)
+		"ugli munkys"         # ugly monkeys (mispronounced)
+		"weeeeek speeeces"    # weak species (mispronounced)
+
+phrase "exile humanika: you inferior individual"
+	word
+		"yoo ugli foo"        # You ugly fool (mispronounced)
+		"skinlis munky"       # Skinless monkey (mispronounced)
+		"ugli halching"       # Ugly hatchling (mispronounced)
+		"frellin' copse"      # Frelling corpse (mispronounced)
```

Comment:
Ah, I see you missed the third one: `"weeeeek speeeces"    # weak species (mispronounced)`

Reply:
It is  now `"yer've humpd corp's" # you have performed an [epithet for having performed an act of procreation] with a corpse (mispronounced)`

Reply:
I went with "humping corpses" because desecrating the bodies of the dead would be one of the most horrible crimes imaginable to a species that relies on the skin of their dead ancestors to pass knowledge to later generations. This would be, by Korath standards, quite an extreme insult.